x/crosschain/keeper/keeper_out_tx_tracker.go

```diff
@@ -167,16 +170,63 @@
 		return nil, zetaObserverTypes.ErrSupportedChains
 	}
 
-	adminPolicyAccount := k.zetaObserverKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_out_tx_tracker)
-	isAdmin := msg.Creator == adminPolicyAccount
+	if msg.Proof == nil { // without proof, only certain accounts can send this message
+		adminPolicyAccount := k.zetaObserverKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_out_tx_tracker)
+		isAdmin := msg.Creator == adminPolicyAccount
 
-	isObserver, err := k.zetaObserverKeeper.IsAuthorized(ctx, msg.Creator, chain)
-	if err != nil {
-		ctx.Logger().Error("Error while checking if the account is an observer", err)
+		isObserver, err := k.zetaObserverKeeper.IsAuthorized(ctx, msg.Creator, chain)
+		if err != nil {
+			ctx.Logger().Error("Error while checking if the account is an observer", err)
+			return nil, sdkerrors.Wrap(zetaObserverTypes.ErrNotAuthorized, fmt.Sprintf("error  IsAuthorized %s", msg.Creator))
+		}
+		// Sender needs to be either the admin policy account or an observer
+		if !(isAdmin || isObserver) {
```

Comment:
This can be refactored a bit to have a condition like
if !(isAdmin || isObserver || isProven)